cli/Cargo.toml

```diff
@@ -36,8 +36,6 @@ dirs = "2.0.2"
 dlopen = "0.1.8"
 futures = { version = "0.3.1", features = [ "compat", "io-compat" ] }
 http = "0.2.0"
-hyper = "0.13.1"
-hyper-rustls = "0.19.0"
```

Comment:
Why? These seems unrelated?

Reply:
Check out this comment: https://github.com/denoland/deno/issues/3607#issuecomment-573410167